add_quants_in_report_menu/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Add Quants Menu In Report
+========================
+
+This module extends the functionality of report to add a Quants menu in report for sale man.
```

Comment:
This module adds a Quants entry in sales menu and for Accounting & Finance group